From c09865aa3af8248c69b7aa064fe6e96f68e03de1 Mon Sep 17 00:00:00 2001
From: Jaroslav Mracek <jmracek@redhat.com>
Date: Fri, 8 Mar 2024 14:06:26 +0100
Subject: [PATCH 4/4] Update documentation of history userinstalled command
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream commit: 929d9133971b53eabfd65d989ded0de8f72f95ea

The described behavior differs from current behavior therefore
it might create a confusion.

Resolves: https://issues.redhat.com/browse/RHEL-15902
Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 doc/command_ref.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/command_ref.rst b/doc/command_ref.rst
index 8991df2f..ffb84407 100644
--- a/doc/command_ref.rst
+++ b/doc/command_ref.rst
@@ -795,8 +795,8 @@ transactions and act according to this information (assuming the
     the current state of RPMDB, it will not undo the transaction.
 
 ``dnf history userinstalled``
-    Show all installonly packages, packages installed outside of DNF and packages not
-    installed as dependency. I.e. it lists packages that will stay on the system when
+    Show all packages installed by user, installed from a group or a module profile, and packages
+    installed outside of DNF. I.e. it lists packages that will stay on the system when
     :ref:`\autoremove_command-label` or :ref:`\remove_command-label` along with
     `clean_requirements_on_remove` configuration option set to True is executed. Note the same
     results can be accomplished with ``dnf repoquery --userinstalled``, and the repoquery
-- 
2.45.0

